### PR TITLE
Vue app initialize

### PR DESCRIPTION
**Pull request by DewDropsDoha**
Fixes #

- [ ] Does not depend on other PRs
- [x] Depends on other PR being merged first: #172

## What kind of change does this PR introduce (check at least one)

- [ ] Bugfix
- [ ] Improvement
- [x] New feature
- [ ] Refactor
- [ ] Infrastructure
- [ ] Other (please specify)

## What was done
Vue app created using vue cli. Bootstrap vue linked with vue app. Simple frontend build.

## How to test

- [ ] Needs regression testing (add QA needed label)

## Deploy risk & follow up

Please assess risk & follow up on deployment of this PR. Elaborate your choice if selecting anything other than 'No risk'.

- [x] No risk
- [ ] Low
- [ ] High

QA - if regression testing was requested:

- [ ] QA has verified

### CODE REVIEW

File: doha-aws-node-project/handler.js [hunk 1]
```diff
@@ -1,15 +1,262 @@
...
-        message: "Go Serverless v2.0! Your function executed successfully!",
-        input: event,
-      },
-      null,
-      2
-    ),
+  let response = {};
+  response["headers"] = {
+    "Access-Control-Allow-Origin": "*",
+    "Access-Control-Allow-Credentials": true,
```
[the remark this answers is not in the document]
Comment (DewDropsDoha) [reply doha-aws-node-project/handler.js]: Unnecessary code removed as cors added in serverless lambda. Commit a97f939

File: doha-aws-node-project/handler.js [hunk 2]
```diff
@@ -1,15 +1,262 @@
...
+          }
+
+          fileName = fileName.replace("uploaded/", "");
+          fileName = fileName.replace("converted/", "");
+          const message = user + action + fileName;
+
+          response = await publishToSns(message);
+          response["headers"] = {
+            "Access-Control-Allow-Origin": "*",
+            "Access-Control-Allow-Credentials": true,
```
[the remark this answers is not in the document]
Comment (DewDropsDoha) [reply doha-aws-node-project/handler.js]: Unnecessary code removed as cors added in serverless lambda. Commit a97f939

File: doha-aws-node-project/vue-app/aws-world/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# aws-world
```
[the remark this answers is not in the document]
Comment (DewDropsDoha) [reply doha-aws-node-project/vue-app/aws-world/README.md]: Removed readme.md inside vue-app. Commit ddd39fc

File: doha-aws-node-project/handler.js [hunk 4]
```diff
@@ -1,15 +1,262 @@
...
+              response["body"] = "File has been resized to 144P";
+            } catch (error) {
+              response["statusCode"] = 400;
+              response["body"] = `Video conversion to HD failed! ${error}`;
+            }
+          } else {
+            response = await saveToDynamoDb(fileNameWithPath);
+            response["headers"] = {
+              "Access-Control-Allow-Origin": "*",
+              "Access-Control-Allow-Credentials": true,
```
[the remark this answers is not in the document]
Comment (DewDropsDoha) [reply doha-aws-node-project/handler.js]: Unnecessary code removed as cors added in serverless lambda. Commit a97f939

File: doha-aws-node-project/vue-app/aws-world/src/components/HelloWorld.vue [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+    },
+    fileUpload($event) {
+      this.form.file = $event.target.files[0];
+      this.form.fileName = $event.target.files[0].name;
+      this.form.fileType = $event.target.files[0].size;
+      this.form.uploadDate = this.getCurrentDate;
+    },
+    onSubmit(event) {
+      event.preventDefault();
+      // alert(JSON.stringify(this.form));
```
[the remark this answers is not in the document]
Comment (DewDropsDoha) [reply doha-aws-node-project/vue-app/aws-world/src/components/HelloWorld.vue]: Removed commented-out code. Commit ed4d93e

File: doha-aws-node-project/vue-app/aws-world/src/components/HelloWorld.vue [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+          .then((response) => {
+            this.statusMessage = "Uploading to S3 Bucket ....."
+            const s3UploadUrl = response.data
+            this.uploadToS3(s3UploadUrl)
+            this.onReset()
+          })
+          .catch(error=>{
+            console.log(error)
+            // this.statusCode = 400
+            // this.statusMessage = "Error getting signed url!"
```
[the remark this answers is not in the document]
Comment (DewDropsDoha) [reply doha-aws-node-project/vue-app/aws-world/src/components/HelloWorld.vue]: Removed commented-out code. Commit ed4d93e

[the remark this answers is not in the document]
Comment (DewDropsDoha) [reply doha-aws-node-project/vue-app/aws-world/src/components/HelloWorld.vue]: These commented-out code I used for testing purpose. Whenever I finished my testing, I didn't remove these codes rather I commented the codes. After task was done, I forgot to remove this and pushed.
Whenever my test is done, I must remove testing codes rather then commenting. Also I must check before push and commit. Double checking costs minutes but definitely saves hours later.

File: doha-aws-node-project/vue-app/aws-world/src/components/HelloWorld.vue [hunk 7]
```diff
@@ -0,0 +1,229 @@
...
+        this.statusMessage = "Upload Successfull!"
+        setTimeout(() => {
+          this.statusCode = null
+          this.statusMessage = ""
+        }, 1000);
+      })
+      .catch(error=>{
+        console.log(error)
+      })
+    },
```
[the remark this answers is not in the document]
Comment (DewDropsDoha) [reply doha-aws-node-project/vue-app/aws-world/src/components/HelloWorld.vue]: Fixed code formatting. Used try catch block. Commit a89beba

File: doha-aws-node-project/vue-app/aws-world/src/components/HelloWorld.vue [hunk 8]
```diff
@@ -0,0 +1,229 @@
...
+    onReset(event) {
```
[the remark this answers is not in the document]
Comment (DewDropsDoha) [reply doha-aws-node-project/vue-app/aws-world/src/components/HelloWorld.vue:218]: Fixed code formatting. Used try catch block. Commit a89beba

**Review by mhazad — APPROVED** [review]
LGTM

**Pull request merged**